pcsx2/pcsx2.vcxproj.filters

```diff
@@ -533,9 +533,6 @@
     <ClCompile Include="IopMem.cpp">
       <Filter>System\Ps2\Iop</Filter>
     </ClCompile>
-    <ClCompile Include="IopSif.cpp">
-      <Filter>System\Ps2\Iop</Filter>
```

Comment:
Isn't this still needed `iopsif`?

Reply:
nope.

Reply:
iopsif probably shouldn't even be there anymore.